.gcloudignore

```diff
@@ -1,17 +0,0 @@
-# This file specifies files that are *not* uploaded to Google Cloud Platform
```

Comment:
Revert deletion of this file?

Reply:
I realized this file is added every time the project is deployed to gcloud, so I was thinking we could exclude it from the repo?

Reply:
Seems prudent to keep it around in the repo then right? Similar to how .gitignore is included.

Reply:
Ok, I'll revert the deletion for it. Just wondering if its better to include the file in the same directory it is automatically generated in (`/frontend`) or the main project directory?

Reply:
Do you know why it generates in the frontend directory? Is it because that's the directory you have to deploy from?

Reply:
Yeah, that's where I deploy from

Reply:
If you deployed from the root would the file be generated there or always under /frontend? If so I'd say go ahead and delete this in favor of one under /frontend

Reply:
Ok, got it. Since it only allows you to deploy from /frontend, I will favor that file and keep this deleted